cabal version 2.19.1 (#1662)
kadena-io/chainweb-node: CHANGELOG.md

# `chainweb-node` Changelog

## 2.19.1 (2023-05-22)

> **NOTE: THIS VERSION REPLACED BY 2.19. PLEASE UPDATE AS SOON AS POSSIBLE.**

Author: ```suggestion
**NOTE: THIS VERSION REPLACES 2.19. PLEASE UPDATE AS SOON AS POSSIBLE.**
```

> **NOTE: THIS VERSION REPLACED BY 2.19. PLEASE UPDATE AS SOON AS POSSIBLE.**

Reviewer: ```suggestion
**NOTE: THIS VERSION SUPERSEDES 2.19. PLEASE UPDATE AS SOON AS POSSIBLE.**
```

This version replaces all previous versions. Node administrators must upgrade to

> this version before that date.

Reviewer: ```suggestion
this version before 2023-06-01T00:00:00Z.
```

This version will expire on 2023-09-07.

To upgrade, pull the latest docker image or download the binary and restart the node.

Changes:

> *   Revert a change in version 2.19 the causes failures in existing smart

Reviewer: ```suggestion
*   Disable user function return value typechecking (#1661)
```

>     contracts on mainnet. (TODO)

Author: ```suggestion
    contracts on mainnet. (#1661)
```

>     contracts on mainnet. (#1661)

Reviewer: ```suggestion

```
